docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Dev Tools Plugins
+description: Learn about using dev tools plugins to inspect and debug your Expo Project.
+---
+
+import { Tab, Tabs } from '~/ui/components/Tabs';
+import { Terminal } from '~/ui/components/Snippet';
+
+> Available with SDK 50 and above.
+
+Dev tools plugins are tools available in your local development environment that help you to debug your app. They typically consist of a small amount of code your add to project that enables two-way communication between your app and an external Chrome window, which can then display tools for inspecting your app, triggering certain behaviors for testing, and more. Dev tools plugins are similar to Flipper plugins, but are available in Expo Go and Development Builds without adding any native modules or config plugins to your project.
+
+## Adding a dev tools plugin to your project
+
+Dev tools plugins are installed as packages in your project. A plugin will typically have a small amount of connecting code that you add to your app in order to enable two-way communication between your app and the plugin. This code will generally be invoked in your app's root component, especially if the plugin is inspecting a module that is available for the entire time your app is running. It is recommended that you wrap this code in a conditional that only runs in development mode, so that it is not included in production builds of your app:
+
+```jsx
+if (__DEV__) {
+  // initialize your DevTools plugin here
+}
+```
+
+In some cases, you may need to interact with a plugin directly. All plugins communicate through exports from `expo/devtools`, and you can send and listen to messages through `connectPluginFromDevToolsAsync`:
+
+```jsx
+import { connectPluginFromDevToolsAsync } from 'expo/devtools';
+
+const client = await connectPluginFromDevToolsAsync();
+
+client.sendMessage('ping', { from: 'app' });
+client.addMessageListener('pong', (data) => {
+  console.log('receiving pong', data);
+});
+```
+
+### Compatibility with Expo Go and Development Builds
+
+Dev tools plugins themselves should only include JavaScript code, meaning they are generally compatible with both Expo Go and [Development Builds](development-builds/introduction/), and it should not require creating a new development build in order to add a dev tools plugin. If the underlying module that the plugin is inspecting is from a package that includes native code that is not already part of Expo Go, then you would need to create a new development build with that package included in order to use both the component and the plugin.
```

Comment:
```suggestion
Dev tools plugins should only include JavaScript code. They are generally compatible with Expo Go and [Development builds](/development-builds/introduction/) and should not require creating a new development build to add the plugin. If a package's underlying module that the plugin inspects includes native code and is not part of Expo Go, create a new development build to use both the component and the plugin from that package.
```

- The last sentence didn't have a structure or a flow that I could understand
- We prefer Development builds
- Also, the link was broken